### PR TITLE
feat: updates and enhancements

### PR DESCRIPTION
**Pull request by nasbench**
This PR covers the following:

- Changed the `related` type of some registry rule to `obsoletes` since the rule linked is already `deprecated`
- Updated metadata (title, description, FP section) of some rules
- Added some FP tuning found in the testing env
- Merged `4e8d5fd3-c959-441f-a941-f73d0cdcdca5` into `73a883d0-0348-4be4-a8d8-51031c2564f8` since they are expressing the same logic
- Renamed a couple of rules to follow the naming convention
- Added new rules related to `renamed office binaries` and `suspicious env variables`
- Other quality of life updates

Check the `Files changed` section for the full list

### CODE REVIEW

File: rules/windows/registry/registry_set/registry_set_suspicious_env_variables.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            - 'TVoAAAAAAAAAAAAA'
+            - 'TVpTAQEAAAAEAAAA'
+            # Base64 Invoke- (UTF-8)
+            - 'SW52b2tlL'
+            - 'ludm9rZS'
+            - 'JbnZva2Ut'
+            # Base64 Invoke- (UTF-16LE)
+            - 'SQBuAHYAbwBrAGUALQ'
+            - 'kAbgB2AG8AawBlAC0A'
+            - 'JAG4AdgBvAGsAZQAtA'
```
Comment (Neo23x0) [inline rules/windows/registry/registry_set/registry_set_suspicious_env_variables.yml:40]:
```suggestion
    selection_details:
        - Details:
            - 'powershell'
            - 'pwsh'
        - Details|contains:
            # Add more suspicious strings in env variables below
            - '\AppData\Local\Temp\'
            - 'C:\Users\Public\'
            # Base64 MZ Header
            - 'TVqQAAMAAAAEAAAA' # MZ..........
            - 'TVpQAAIAAAAEAA8A'
            - 'TVqAAAEAAAAEABAA'
            - 'TVoAAAAAAAAAAAAA'
            - 'TVpTAQEAAAAEAAAA'
            # Base64 Invoke- (UTF-8)
            - 'SW52b2tlL'
            - 'ludm9rZS'
            - 'JbnZva2Ut'
            # Base64 Invoke- (UTF-16LE)
            - 'SQBuAHYAbwBrAGUALQ'
            - 'kAbgB2AG8AawBlAC0A'
            - 'JAG4AdgBvAGsAZQAtA'
        - Details|startswith:  # https://gist.github.com/Neo23x0/6af876ee72b51676c82a2db8d2cd3639
            - 'SUVY'
            - 'SQBFAF'
            - 'SQBuAH'
            - 'cwBhA'
            - 'aWV4'
            - 'aQBlA'
            - 'R2V0'
            - 'dmFy'
            - 'dgBhA'
            - 'dXNpbm'
            - 'H4sIA'
            - 'Y21k'
            - 'cABhAH'
            - 'Qzpc'
            - 'Yzpc'
```

**Review by Neo23x0 — APPROVED** [review]

File: rules/windows/registry/registry_set/registry_set_suspicious_env_variables.yml (hunk 1 above)
Comment (Neo23x0) [reply rules/windows/registry/registry_set/registry_set_suspicious_env_variables.yml]: Shouldn't the different Details be ORed? I mean, a TVqQAA in an ENV variable is suspicious enough. It doesn't require an additional powershell or pwsh string. In an env var these strings alone should be suspicious enough, I'd say.

Comment (Neo23x0) [reply rules/windows/registry/registry_set/registry_set_suspicious_env_variables.yml]: I've also added some strings from my cheat sheet. Adding them with "Details|startswith" should be specific enough.

Comment (nasbench) [reply rules/windows/registry/registry_set/registry_set_suspicious_env_variables.yml]: Yes, they should be ORed you're right. I missed the `-` for that.

Comment (nasbench) [reply rules/windows/registry/registry_set/registry_set_suspicious_env_variables.yml]: Also great additions. The `startswith` is the better modifier for cases like those

File: rules-deprecated/windows/proc_creation_win_mavinject_proc_inj.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        CommandLine|contains: ' /INJECTRUNNING '
+    condition: selection
+falsepositives:
+    - Unknown
+level: high
+tags:
```
Comment (frack113) [inline rules-deprecated/windows/proc_creation_win_mavinject_proc_inj.yml:22]: just to keep the order logic tags before logsource ans level at the end

File: rules/windows/process_creation/proc_creation_win_create_link_osk_cmd.yml [hunk 3]
```diff
@@ -16,13 +16,13 @@ logsource:
 detection:
     selection_img:
         - Image|endswith: '\cmd.exe'
-        - OriginalFileName: 'Cmd.exe'
+        - OriginalFileName: 'Cmd.Exe'
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_create_link_osk_cmd.yml:19]: I kown it should not change the detection , but for me `modified` is need as the detection section is edited.

File: rules/windows/process_creation/proc_creation_win_renamed_office_processes.yml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+            - 'OneNote.exe'
+            - 'POWERPNT.EXE'
+            - 'WinWord.exe'
+        - Description:
+            - 'Microsoft Access'
+            - 'Microsoft Excel'
+            - 'Microsoft OneNote'
+            - 'Microsoft PowerPoint'
+            - 'Microsoft Word'
+        - Product|contains: Microsoft Office'
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_renamed_office_processes.yml:28]:
```suggestion
        - Product|contains: 'Microsoft Office'
```
Other tools of the office suite don't have  the same product ?

File: tests/test_rules.py [hunk 5]
```diff
@@ -65,6 +65,7 @@ def get_rule_yaml(self, file_path: str) -> dict:
         # "There are rule files with extensions other than .yml")
 
     def test_legal_trademark_violations(self):
+        # See Issue # https://github.com/SigmaHQ/sigma/issues/1014
```
Comment (frack113) [inline tests/test_rules.py:68]: 1014 is Develop Sigma rules for Living Off The Land Binaries and Scripts ???

Comment (Neo23x0) [reply tests/test_rules.py]: fixed the link

**Review by frack113 — APPROVED** [review]

**Pull request merged**